packages/@aws-cdk/aws-dynamodb/lib/table.ts

```diff
@@ -76,6 +124,30 @@ export class Table extends Construct {
         return this;
     }
 
+    public addReadAutoScaling(props: DynamoDBAutoScalingProps) {
+        this.validateDynamoDBAutoScalingProps(props);
+        const readAutoScalingRole = this.buildAutoScalingRole('ReadAutoScalingRole');
+
+        const readScalableTargetResource = new applicationautoscaling.ScalableTargetResource(
```

Comment:
This is not ideal. Usually we don't like to use "foreign" low-level resources. The "right way" would have been to implement a high level construct for applicationautoscaling and then use it here. Can you please just open an issue for us to remember to migrate this code once we do have an L2 for appautoscaling?

Reply:
created issue https://github.com/awslabs/aws-cdk/issues/640